backend/api/Controllers/SyncMayanController.cs

```diff
@@ -0,0 +1,78 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Api.Models.Mayan.Sync;
+using Pims.Api.Services;
+using Pims.Dal.Entities;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Pims.Api.Controllers
+{
+    /// <summary>
+    /// SyncMayanController class, provides endpoints to handle document requests.
+    /// </summary>
+    //[Authorize]
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/documents/")]
+    [Route("/documents")]
+    public class SyncMayanController : ControllerBase
+    {
+        #region Variables
+        private readonly IDocumentSyncService _documentSyncService;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a SyncMayanController class.
+        /// </summary>
+        /// <param name="documentSyncService"></param>
+        public SyncMayanController(IDocumentSyncService documentSyncService)
+        {
+            _documentSyncService = documentSyncService;
+        }
+        #endregion
+
+        #region Endpoints
+
+        /// <summary>
+        /// Uploads the passed document.
+        /// </summary>
+        [HttpPatch("sync/mayan/documenttype")]
+        //[HasPermission(Permissions.PropertyAdd)] // TODO: put the correct permission
```

Comment:
Yes, is there a task for this yet? this would be a bad one to forget.

Reply:
Not that I know of. Could be covered by the document claims one